app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,8 @@
+class TasksController < ApplicationController
+
+    def index
+        tasks = Task.all
+        render json: jobs
```

Comment:
Looks like your variable is named tasks but your are rendering jobs

Reply:
smh...thank you :) I fixed before merging.